lib/common/l2p.h

```diff
@@ -140,9 +141,10 @@ l2p_create(void)
     for (i = 0; i < RTE_MAX_ETHPORTS; i++)
         l2p->ports[i].pid = i;
 
-    /* Set them to stopped first. */
-    for (i = 0; i < RTE_MAX_LCORE; i++)
-        l2p->stop[i] = 1;
+    for (i = 0; i < RTE_MAX_LCORE; i++) {
+        l2p->started[i] = 0;
+        l2p->stopped[i] = 0;
+    }
```

Comment:
The setting of started and stopped to zero is not required as calloc() above would have handled it. Plus I see on line 136 I did memset the structure and 135-136 is not required as well.

Reply:
Love the comment in line 135.